src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -141,5 +144,22 @@ public void PressDialogButton(string dialogAutomationName, string buttonAutomati
 
         public void DialogSendKeys(string dialogAutomationName, string keys)
             => _inProc.DialogSendKeys(dialogAutomationName, keys);
+
+        public void NavigateToSendKeys(string keys)
+            => _inProc.SendKeysToNavigateTo(keys);
+
+        public void WaitForActiveView(string viewName)
+        {
+            while (true)
+            {
+                System.Threading.Thread.Sleep(100);
```

Comment:
There's some generic helper for "retry until" pattern that we should reuse. That way we can put timeout logic in one place.